pymatgen/command_line/bader_caller.py

```diff
@@ -531,8 +527,9 @@ def _get_filepath(filename: str, msg: str = "") -> str | None:
         return paths[0]
 
     chgcar_path = _get_filepath("CHGCAR", "Could not find CHGCAR!")
-    if chgcar_path is not None:
-        chgcar = Chgcar.from_file(chgcar_path)
+    if chgcar_path is None:
+        raise FileNotFoundError("Could not find CHGCAR!")
```

Comment:
The `_get_filepath` method already issued a warning in line 529, maybe this is not necessary?

Reply:
i guess `Chgcar.from_file(chgcar_path)` will already raise if the file is missing?

Reply:
Maybe no... By `Chgcar.from_file`
https://github.com/materialsproject/pymatgen/blob/d07164fd27b955010ad411033d262ef3d935d09c/pymatgen/io/vasp/outputs.py#L3661-L3672

And then `VolumetricData.parse_file`:
https://github.com/materialsproject/pymatgen/blob/d07164fd27b955010ad411033d262ef3d935d09c/pymatgen/io/vasp/outputs.py#L3431-L3535

Then`zopen`: https://github.com/materialsvirtuallab/monty/blob/4d60fd4745f840354e7b3f48346eaf3cd68f2b35/monty/io.py#L19-L45

But certainly `open` would complain (maybe not so descriptive). But `_get_filepath` would issue a warning anyway, so maybe don't bother issure another here 😄

Reply:
By the way, I added the `if chgcar_path not None` check to avoid `mypy` error, because `_get_filepath` could return `None`